nnpdf_data/nnpdf_data/new_commondata/LHCB_Z0J_13TEV_2022/kinematics_dimuon_pT.yaml

```
@@ -0,0 +1,169 @@
+bins:
+- pT:
+    min: 0.0
+    mid: 1.1
+    max: 2.2
+  sqrts:
+    min: null
+    mid: 13
```

```suggestion
    mid: 13
```

ah, gotcha! The problem you have is that you are using sqrts in TeV, if you write 13000 that should solve the problem you're seeing.

The same is probably true in The rapidity case as well. Make sure that the units are always GeV.

---

Yes, really my bad. I've fixed that and now points are all within 1. Thanks for spotting it!